src/lib/modules/Relations/backoffice/components/RelationRemover/RelationRemover.js

```diff
@@ -39,10 +39,16 @@ export default class RelationRemover extends Component {
       >
         <Modal.Header>Confirm removal</Modal.Header>
         <Modal.Content>
-          Are you sure you want to delete this relation?
+          <p>Are you sure you want to remove this relation?</p>
+          {related.pid_type === 'docid' && (
```

Comment:
This is the correct version (pseudocode):
```suggestion
          const isSibling = ['language', 'edition', 'other'].includes(relationType);
          {isSibling && (
```
Have a look for example [here](https://github.com/inveniosoftware/react-invenio-app-ils/blob/master/src/lib/api/relations/relations.js#L18).